docs_guidelines/how_to/crossreference.rst

```diff
@@ -0,0 +1,89 @@
+================================================
+Crossreference to other pages from documentation
```

Comment:
`intersphinx` seems to work flawlessly. Can you add a section about how to use that here? I think a good example would be referencing a `qiskit-terra` class in `qiskit-nature` doc.

Reply:
Would that fit better here or in the external links guide?

Reply:
I think it fit better here.

Reply:
But you can also do it like the jupyter notebook link. Add a note here and add the main text there or the reverse.